### PR TITLE
Create Disable Free Worlds.yaml

### PR DESCRIPTION
**Pull request by ziproot**

### CODE REVIEW

**Review by TheGiraffe3 — APPROVED** [review]
~~but why?~~

**Comment by ziproot** [comment]
https://steamcommunity.com/id/alison/recommended/404410/

**Comment by TheGiraffe3** [comment]
Ah, makes sense. Thanks.

File: manifests/Disable Free Worlds.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Disable Free Worlds
+authors: ziproot
+license: CC0
```
Comment (MCOfficer) [inline manifests/Disable Free Worlds.yaml:3]:
```suggestion
license: CC0-1.0
```

**Pull request merged**